src/ModuleApi.ts

```diff
@@ -46,14 +47,14 @@ export interface ModuleApi {
 
     /**
      * Opens a dialog in the client.
-     * @param title The title of the dialog
+     * @param moduleUiDialogProps Module UI Dialog props.
```

Comment:
I'm concerned that this is a breaking change: it will cause an incompatibility with any existing modules that expect to be able to pass a string here.

It would be better to allow this to be *either* a string *or* a `ModuleUiDialogProps`, with a typescript overload signature.

Also, the comment here isn't very useful:

```suggestion
     * @param moduleUiDialogProps Options for the new dialog.
```

Reply:
I changed it to be non-breaking and accept both a string for the title or an object for more detailed configurations.